### DISCUSSION TITLE
Multiple paginators causes unit test failed

### DISCUSSION THREAD

**Discussion by jacksontong** [livewire/livewire#4190] [7. Bugs (archived)]
We have been using version v2.4.4 which doesn't have #3684, and today we upgrade to v2.7.2 our unit tests failed with this error
```
Next Facade\Ignition\Exceptions\ViewException: 
Using $this when not in object context (View: vendor/livewire/livewire/src/views/pagination/tailwind.blade.php) 
in vendor/livewire/livewire/src/views/pagination/tailwind.blade.php:3
```
it works fine on the browser.

if we remove the pagination from the test, it would work
```php
@if(!app()->runningUnitTests())
    {{ $folders->links() }}
@endif
```

Another strange thing is that it doesn't fail for a single test `php artisan test --filter test_pagination`, but if we run a full test `php artisan test` it fails again

**Comment by joshhanley** [comment]
@jacksontong do you publish your pagination views? If so, you will need to publish them again, to get the latest changes.

Hope this helps!

**Reply by jacksontong** [reply]
I've published the pagination views but still same error.
```bash
php artisan livewire:publish
ls resources/views/vendor/livewire
//bootstrap.blade.php  simple-bootstrap.blade.php  simple-tailwind.blade.php  tailwind.blade.php
```

**Reply by joshhanley** [reply]
@jacksontong ah ok, thanks for confirming. Hmm I wonder what is going on.

Any chance you could create a small example repo which reproduces the issue with as little code as possible so we can investigate it?

Thanks!
